CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
+- to designate important Creator Components, Objets, etc. Example: __Scene__, __Sprite Component__
+- to designate important Creator UI elements. Example: __Hierarchy Panel__, __Assets Panel__
+
+### Using Italics
+- list items. Example: *apple*, *orange*, *banana*, and *pear*
```

Comment:
Maybe a bold here would be fine? I think it's better not to have too many types of styles?